diff --git a/node_modules/expo-dev-menu/ios/SwiftUI/CustomItems.swift b/node_modules/expo-dev-menu/ios/SwiftUI/CustomItems.swift
index b709262..1737d99 100644
--- a/node_modules/expo-dev-menu/ios/SwiftUI/CustomItems.swift
+++ b/node_modules/expo-dev-menu/ios/SwiftUI/CustomItems.swift
@@ -21,7 +21,7 @@ struct CustomItems: View {
               .frame(maxWidth: .infinity, alignment: .leading)
           }
           .padding()
-          .background(Color.expoSecondarySystemBackground)
+          .background(Color(UIColor.secondarySystemBackground))
           .clipShape(RoundedRectangle(cornerRadius: 12))
         }
       }
diff --git a/node_modules/expo-dev-menu/ios/SwiftUI/DevMenuAppInfo.swift b/node_modules/expo-dev-menu/ios/SwiftUI/DevMenuAppInfo.swift
index 35fa44f..275e6f4 100644
--- a/node_modules/expo-dev-menu/ios/SwiftUI/DevMenuAppInfo.swift
+++ b/node_modules/expo-dev-menu/ios/SwiftUI/DevMenuAppInfo.swift
@@ -43,7 +43,7 @@ struct DevMenuAppInfo: View {
           .disabled(viewModel.clipboardMessage != nil)
         }
       }
-      .background(Color.expoSystemBackground)
+      .background(Color(UIColor.systemBackground))
       .cornerRadius(18)
     }
   }
diff --git a/node_modules/expo-dev-menu/ios/SwiftUI/DevMenuButtons.swift b/node_modules/expo-dev-menu/ios/SwiftUI/DevMenuButtons.swift
index 3850cb4..cc20b4b 100644
--- a/node_modules/expo-dev-menu/ios/SwiftUI/DevMenuButtons.swift
+++ b/node_modules/expo-dev-menu/ios/SwiftUI/DevMenuButtons.swift
@@ -30,7 +30,7 @@ struct DevMenuActionButton: View {
       .padding()
     }
     .disabled(disabled)
-    .background(Color.expoSecondarySystemBackground)
+    .background(Color(UIColor.secondarySystemBackground))
     .opacity(disabled ? 0.6 : 1.0)
   }
 }
@@ -69,7 +69,7 @@ struct DevMenuToggleButton: View {
       .disabled(disabled)
     }
     .padding()
-    .background(Color.expoSecondarySystemBackground)
+    .background(Color(UIColor.secondarySystemBackground))
     .opacity(disabled ? 0.6 : 1.0)
   }
 }
diff --git a/node_modules/expo-dev-menu/ios/SwiftUI/DevMenuDeveloperTools.swift b/node_modules/expo-dev-menu/ios/SwiftUI/DevMenuDeveloperTools.swift
index 84fbef8..5b964f8 100644
--- a/node_modules/expo-dev-menu/ios/SwiftUI/DevMenuDeveloperTools.swift
+++ b/node_modules/expo-dev-menu/ios/SwiftUI/DevMenuDeveloperTools.swift
@@ -48,7 +48,7 @@ struct DevMenuDeveloperTools: View {
           disabled: !(viewModel.devSettings?.isHotLoadingAvailable ?? true)
         )
       }
-      .background(Color.expoSystemBackground)
+      .background(Color(UIColor.systemBackground))
       .cornerRadius(18)
     }
   }
diff --git a/node_modules/expo-dev-menu/ios/SwiftUI/DevMenuRNDevMenu.swift b/node_modules/expo-dev-menu/ios/SwiftUI/DevMenuRNDevMenu.swift
index 0c24e7e..fe2e4e8 100644
--- a/node_modules/expo-dev-menu/ios/SwiftUI/DevMenuRNDevMenu.swift
+++ b/node_modules/expo-dev-menu/ios/SwiftUI/DevMenuRNDevMenu.swift
@@ -12,7 +12,7 @@ struct DevMenuRNDevMenu: View {
         .foregroundColor(.primary)
         .frame(maxWidth: .infinity, alignment: .leading)
     }
-    .background(Color.expoSecondarySystemBackground)
+    .background(Color(UIColor.secondarySystemBackground))
     .cornerRadius(18)
   }
 }
diff --git a/node_modules/expo-dev-menu/ios/SwiftUI/HeaderView.swift b/node_modules/expo-dev-menu/ios/SwiftUI/HeaderView.swift
index 90f7aec..73c9552 100644
--- a/node_modules/expo-dev-menu/ios/SwiftUI/HeaderView.swift
+++ b/node_modules/expo-dev-menu/ios/SwiftUI/HeaderView.swift
@@ -31,9 +31,9 @@ struct HeaderView: View {
         ZStack {
           Circle()
           #if os(tvOS)
-            .fill(Color.expoSystemGray4.opacity(0.2))
+            .fill(Color(UIColor.systemGray4).opacity(0.2))
           #else
-            .fill(Color.expoSystemGray6)
+            .fill(Color(UIColor.systemGray6))
           #endif
             .frame(width: 36, height: 36)
 
diff --git a/node_modules/expo-dev-menu/ios/SwiftUI/HostUrl.swift b/node_modules/expo-dev-menu/ios/SwiftUI/HostUrl.swift
index 8f60116..7914dd7 100644
--- a/node_modules/expo-dev-menu/ios/SwiftUI/HostUrl.swift
+++ b/node_modules/expo-dev-menu/ios/SwiftUI/HostUrl.swift
@@ -27,7 +27,7 @@ struct HostUrl: View {
           .foregroundColor(.secondary.opacity(0.7))
       }
       .padding()
-      .background(Color.expoSecondarySystemBackground)
+      .background(Color(UIColor.secondarySystemBackground))
       .cornerRadius(20)
     }
     .buttonStyle(.plain)
